build_targets

```diff
@@ -14,6 +15,8 @@ m52xq user yes yes
 mars userdebug yes yes
 mayfly userdebug yes yes
 miatoll userdebug yes yes
+nx_tab userdebug yes yes
```

Comment:
do validate dependencies please
https://github.com/crdroidandroid/android_device_nintendo_nx_tab/blob/14.0/crdroid.dependencies#L3

Reply:
honest mistake on my part -- should be fixed now